### DISCUSSION TITLE
OSNR and GSNR measurement

### DISCUSSION THREAD

**Discussion by caffery-chen** [Telecominfraproject/oopt-gnpy#447] [Q&A] [answered]
Dear contributor,

I would love to ask one question regarding the OSNR and GSNR measurement approach mentioned in your paper "GNPy: an open source application for physical layer aware open optical networks".

In the field, we usually measure the OSNR for WDM transmission using On-Off OSNR measurement method.
GSNR is measured via the transponder BER vs OSNR B2B curve which is reasonable.

However, when the CUT is turned off, is what we got from OSA the real OSNR value? Is there any NLI power in the inband noise?

I am thinking this is only valid in terms of high dispersion fiber (e.g. SSMF) when only SCI, XCI are dominant and MCI is negligible. Thus, when the CUT is turned off, the NLI vanishes such that from OSA only inband ASE noise left.

I hope my understanding is correct. If not, please advise.

Thanks.

**Comment by jktjkt** [comment]
Cc: @aleFerrari @AndreaDAmico

**Comment by AndreaDAmico** [comment]
Dear @caffery-chen,

Your understanding is completely correct. Let me just add a bit of information. 

The GN model implementation within GNPy include only the SC and XCI. So, depending on the fiber dispersion, if the MCI creates significant issues in the ON-OFF OSNR  evaluation, the NLI estimation provided by GNPy is not accurate as well. 
Nevertheless, we did not observe significant loss in GNPy accuracy up to 2 ps  / (nm km) fiber dispersion.

Additionally, even if the MCI would have been completely absent, the ON-OFF OSNR  evaluation does not provide the "real" value of OSNR as the EDFA responses depend on the spectral load. 
Therefore, when performing these validations, we investigate systems where the ON-OFF OSNR  evaluation provides an adequate evaluation of the  "real" value of OSNR.

**Comment by caffery-chen** [comment]
Dear @AndreaDAmico,

thanks for your explanation on the OSNR evaluation. 

The GSNR is given based on a Coherent transceiver BtB BER vs OSNR value. It is stated in the paper that "This conversion also has the benefit of decorrelating the measurements to the specific characteristics of each distinct TRX."

Not sure if this is true in general according to your experience? We still worried that some market available DSP showing high CD, PDL, SOP penalty might not be able to work although the estimated GSNR via GGN model shows the feasibility.

Besides, am I right that "the span-by-span coherency dependence upon fiber variety and spectral characteristics" is currently not included in the GNpy?

Thanks.

**Reply by AndreaDAmico** [reply, marked as answer]
Dear @caffery-chen,

In the article, the authors meant that the GSNR of the optical line system does not depend on the specific transceiver, which is true in general, and the decorrelation refers to the separation between the optical line system GSNR and the additional SNR degradation due to the specific transceiver (e.g. the electrical noise).
Beside the GSNR degradation, GNPy accumulates also additional metrics that can be used to evaluate additional penalties knowing the transceiver specifics; to date the PMD and CD are already implemented and the PDL will be included in future versions.
In my experience I analysed data from single optical line systems and I have never observed significant penalties in these scenarios. I guess that in more complex use cases these penalties can be significant (e.g. when multiple ROADMs are crossed by the Lightpath).
Yes, "the span-by-span coherency dependence upon fiber variety and spectral characteristics" is currently not included in the GNPy, a more detailed answer can be find in the discussion #453.
